### DISCUSSION TITLE
Ready for testing on Gnome 44

### DISCUSSION THREAD

**Discussion by divsmart** [ddterm/gnome-shell-extension-ddterm#435] [General]
My computer is staying on Silverblue 37 until your amazing extension is Gnome 44-ready.
I am not so computer literate, but I am always trying to find ways to do things with the terminal, so I use ddterm a lot.  
How do I know when it is 44-ready?

**Comment by amezin** [comment]
Current master branch is already compatible with GNOME 44, you can find a download link in https://github.com/ddterm/gnome-shell-extension-ddterm/issues/430#issuecomment-1520660672
New release will be on extensions.gnome.org soon
